yjit/src/stats.rs

```diff
@@ -411,6 +413,7 @@ make_counters! {
     compiled_blockid_count,
     compiled_block_count,
     compiled_branch_count,
+    compile_time_ns,
```

Comment:
Is it `compile_time_ms` or `compile_time_ns` ? 😅

Reply:
Ok I see you're doing the conversion. Was a bit confused because the recorded stat is not the same as reported, but it makes sense 👍